prime-router/settings/staging/0162-la-phl-staging-new-rest-setting.yml

```diff
@@ -1,104 +1,49 @@
-# To submit this to localhost, run:
```

Comment:
We move all of STLTs setting to new location in settings.  In your case, it should be:  ./settings/STLTs/FEXION/la-phl.yml